pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
```

Comment:
Hm, something is messed up. This shouldn't be part of the diff. Can you rebase on master or merge it into your branch?